libs/libarchopenfpga/src/bitstream_setting.h

```diff
@@ -162,6 +173,10 @@ class BitstreamSetting {
   vtr::vector<BitstreamInterconnectSettingId, std::string>
     interconnect_default_paths_;
   std::vector<NonFabricBitstreamSetting> non_fabric_;
+  vtr::vector<OverwriteBitstreamId, OverwriteBitstreamId>
```

Comment:
I see your concern here. Use the vtr::vector<> to store ids is not a good idea. It consumes additional memory and become worse when the size of vector is large. Therefore, in my original comment, I suggest to use std::vector inside. You just force to use a strong id in accessor functions, e.g., ``overwrite_bit_path()``. 

So, you may refer to

https://github.com/lnis-uofu/OpenFPGA/blob/4a07a32902c1ce69ea083b9fe60b1910beb58b78/libs/libfpgabitstream/src/bitstream_manager.h#L267-L268

 https://github.com/lnis-uofu/OpenFPGA/blob/4a07a32902c1ce69ea083b9fe60b1910beb58b78/libs/libfpgabitstream/src/bitstream_manager.cpp#L28-L34

When the invalid_ids_ is typically empty, you will not see any overhead in most cases

Reply:
I will stick to the implementation exactly same as "pb" and "interconnect". Do not plan to make further change